.github/workflows/test-module.yml

```diff
@@ -45,8 +44,23 @@ jobs:
           role-session-name: ${{ github.event.client_payload.pull_request.head.sha || github.sha }}
           aws-region: us-east-1
           # 21600 seconds == 6 hours
+          # 1800 seconds == 30 minutes
           role-duration-seconds: 21600
```

Comment:
Might just be missing it, but what is the addition of this comment? Looks like the role-duration-seconds is still 21600?

Reply:
So on most other Terraform modules, the timeout is set for 30 minutes because the pipelines are significantly shorter.